engine/apps/slack/alert_group_slack_service.py

```diff
@@ -42,28 +45,22 @@ def update_alert_group_slack_message(self, alert_group: "AlertGroup") -> None:
                 blocks=alert_group.render_slack_blocks(),
             )
             logger.info(f"Message has been updated for alert_group {alert_group.pk}")
-        except SlackAPIRateLimitException as e:
+        except SlackAPIRatelimitError as e:
             if alert_group.channel.integration != AlertReceiveChannel.INTEGRATION_MAINTENANCE:
                 if not alert_group.channel.is_rate_limited_in_slack:
                     alert_group.channel.start_send_rate_limit_message_task(e.retry_after)
                     logger.info(
                         f"Message has not been updated for alert_group {alert_group.pk} due to slack rate limit."
                     )
             else:
-                raise e
-
-        except SlackAPIException as e:
-            if e.response["error"] == "message_not_found":  # message deleted from channel
-                logger.info(f"Skip updating slack message for alert_group {alert_group.pk} due message_not_found")
-            elif e.response["error"] == "is_inactive":  # deleted channel error
-                logger.info(f"Skip updating slack message for alert_group {alert_group.pk} due to is_inactive")
-            elif e.response["error"] == "account_inactive":
-                logger.info(f"Skip updating slack message for alert_group {alert_group.pk} due to account_inactive")
-            elif e.response["error"] == "channel_not_found":
-                logger.info(f"Skip updating slack message for alert_group {alert_group.pk} due to channel_not_found")
-            else:
-                raise e
-        logger.info(f"Finished _update_slack_message for alert_group {alert_group.pk}")
```

Comment:
Removing logging here as errors are already logged in [`SlackClient.api_call`](https://github.com/grafana/oncall/blob/091792cdad2fd8235ad3298341f5e31ecd2c9d1c/engine/apps/slack/client.py#L128-L138)